oak_crypto/proto/v1/crypto.proto

```diff
@@ -46,6 +46,7 @@ message EncryptedResponse {
 message AeadEncryptedMessage {
   bytes ciphertext = 1;
   bytes associated_data = 2;
+  bytes nonce = 3;
```

Comment:
Can you add a comment explaining how this is backwards compatible? In particular, that this is currently written to but completely unused for decryption (IIUC).

Reply:
Done